### DISCUSSION TITLE
Is it possible to customise field type or include validation in a field type?

### DISCUSSION THREAD

**Discussion by she-WritesCode** [strapi/strapi#8416] [Legacy Topics]
Is it possible to customise field type? Let's say text field, I need to make sure the user input is a valid Bible chapter or verse. I intend to use [chapter-and-verse](https://github.com/danday74/chapter-and-verse) but not sure where to make it happen.

Also how can I create new field type using strapi?

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
